messenger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .settings import *
```

Comment:
Why are these all separate files using relative imports? It means you now can't run them as a script, which is hugely inconvenient.